src/frontend/js/components/SaleTunnel/SaleTunnelSavePaymentMethod/index.tsx

```diff
@@ -90,23 +88,18 @@ const SaleTunnelSavePaymentMethod = () => {
 
   useEffect(() => {
     if (!payment) {
```

Comment:
`payment` should not be a dependency ?

Reply:
Nope, we just want to trigger this effect when creditCard items has changed.